storage/Utils/Lock.cc

```diff
@@ -33,7 +33,7 @@
 #include "storage/Utils/Lock.h"
 
 
-#define LOCKDIR "/run/libstorage"
+#define LOCKDIR "/run/libstorage-ng"
```

Comment:
AFAIR: Directories can be owned by several RPMs. The lock-file itself is generated if missing so it could be marked as ghost in both RPMs. On the other hand I didn't use RPM much in the past and definitely don't know all the packaging rules.